### PR TITLE
Removed var as per ES6 format.

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
#### Motivation

Replacing usage of var with const or let as appropriate
#### Supporting details

[[https://docs.google.com/spreadsheets/d/1sl6gT8o-7yRPymZprwn6PNBRLQ6KNsKPFILLntsL7gk/edit#gid=0](https://github.com/mdn/content/pull/url)](url)
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/authenticatorassertionresponse/index.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ None.
...
 ```js
-var options = {
+const options = {
   challenge: new Uint8Array([/* bytes sent from the server */])
 };
 
 navigator.credentials.get({ "publicKey": options })
     .then(function (credentialInfoAssertion) {
-    var assertionResponse = credentialInfoAssertion.response;
+    let assertionResponse = credentialInfoAssertion.response;
```
Comment (wbamberg) [inline files/en-us/web/api/authenticatorassertionresponse/index.md:48]:
```suggestion
    const assertionResponse = credentialInfoAssertion.response;
```

I think?

File: files/en-us/web/api/audiotrack/enabled/index.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ This example switches between the main and commentary audio tracks of a media el
 
 ```js
 function swapCommentaryMain() {
-  var videoElem = document.getElementById("main-video");
-  var audioTrackMain;
-  var audioTrackCommentary;
+  const videoElem = document.getElementById("main-video");
+  const audioTrackMain;
+  const audioTrackCommentary;
```
Comment (wbamberg) [inline files/en-us/web/api/audiotrack/enabled/index.md:42]:
```suggestion
  let audioTrackMain;
  let audioTrackCommentary;
```

(you need to initialize `const` declarations right away.)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR! I had a couple of comments.

Also, please don't use "Fixes #16662" in the description, as that will auto-close the issue, while this is only part of the fix.

**Comment by rdotterer09** [comment]
-----BEGIN PGP PUBLIC KEY BLOCK----- 

Version: BCPG C# v1.7.0.0 

 

mI0EYpOu/AEEAL9AdcL31kjx1AbaGM192ac24LGufzMUEeZ4HOGIEsaC289pGb5X 

v6hTVyv1x0BCRP5Oo5X+8SzQukHum8RIpaG7pGJUlbZpKXjcYdQOY7Rddp54+w/H 

QPVswQqK6G5slm4UjONAMjjF4Nfsy6WrF2CoZc/1DcuQ/LkwxuNETdX9ABEBAAG0 

RTI4OTk5MDM0NTgyMTc2NzUxMjg2NjM3ODk0NDI1NTYxMjA0MTE2NTYxMjAzNSA8 

cmRvdHRlcmVyMDlAZ21haWwuY29tPoicBBABAgAGBQJik678AAoJEE1nDVrwIvQr 

+eIEAJiVeBVo5ledTWEhA1CXCxqb1hECZgfrBYAVJZBZKvBCJbuwlC1hwCRuDmzq 

G8eI6mcFNXL/WdMhfqgHnIAOsLEycXyQzcnV7h6GO5kMcOHmAU9KMOHAUPgf06u8 

I5hogL0YU86t475teoS17v0Z2bwTAlBlr5DW4XTLZZ4gKVPo 

=LISa 

-----END PGP PUBLIC KEY BLOCK-----

**Review by teoli2003 — APPROVED** [review]
👍

**Review by wbamberg — APPROVED** [review]
👍 thank you @PrajwalBorkar !

**Comment by PrajwalBorkar** [comment]
> 👍 thank you @PrajwalBorkar !

I will fix the remaining issues also from the sheet.😄

**Pull request merged**